### DISCUSSION TITLE
Get currently selected node/url

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [ueberdosis/tiptap#1342] [General Questions] [answered]
I'm setting up links with TipTap v2. [The example shown in the docs](https://www.tiptap.dev/api/marks/link/) simply uses the browser's `prompt` to get the user's inputs. However, if there's already a link at the current selection, it would be great to prefill the prompt with the current url.

Is there a way to get the current node or it's attributes?

**Comment by philippkuehn** [comment, marked as answer]
You can use `editor.getAttributes('link')`

**Reply by Nick-Mazuk** [reply]
Thanks @philippkuehn, it works perfectly!

Looks like that command isn't in the docs yet, so I'm going to open up a PR (https://github.com/ueberdosis/tiptap/pull/1345) to add it if you don't mind.
